pilz_teleoperation/package.xml

```diff
@@ -18,11 +18,13 @@
 
   <build_depend>message_generation</build_depend>
   <build_depend>std_msgs</build_depend>
+  <build_depend>visualization_msgs</build_depend>
```

Comment:
shouldn't it be run or even test depend? We are not generating any messages with it, but only use it in the demo_app don't we?